### ISSUE TITLE
Reverse sort order

### ISSUE THREAD

**Issue by icp1994** [linuxdeepin/dde-file-manager#46]
Sort order is currently in descending...

![deepinscreenshot20170612182135](https://user-images.githubusercontent.com/15713555/27034719-67e6640a-4f9c-11e7-8b76-baa42a292028.png)

...and in the subfolders too(as expected)

Is there a way to get the reverse sort order?

Freshly installed Manjaro Deepin

```
➔ uname -r
4.9.30-1-MANJARO

```

**Comment by oberon2007** [comment]
Yes, it's really odd. The default sorting should not be z-a but the other way around... 😉

**Comment by leandrofranca** [comment]
Here with 4.11.6-3-ARCH I'm having the same issue.
